ui/core/src/model/dashboard.ts

```diff
@@ -32,7 +32,7 @@ export interface DashboardResource {
 export interface DashboardSpec {
   datasource: DatasourceSelector;
   duration: DurationString;
-  variables: Record<string, AnyVariableDefinition>;
-  panels: Record<string, AnyPanelDefinition>;
+  variables: Record<string, VariableDefinition>;
   layouts: LayoutDefinition[];
+  panels: Record<string, PanelDefinition>;
```

Comment:
Yeah, I remember that in the original spec. I remember discussing it when we did the change to support fancier CSS grid layouts and I think that's when I changed this so that `layouts` is now basically `entrypoint`.